prefab/curl-ssl/README.md

```diff
@@ -1,68 +1,4 @@
-# curl-ssl
+Note:
+   This directory is deprecated and moved into
+   https://github.com/android/ndk-samples/tree/main/prefab/curl-ssl/dependency-integration.
```

Comment:
This URL is wrong. I think this might also get formatted as a code block?

Reply:
fixed, thanks!